package.json

```diff
@@ -9,79 +9,70 @@
     "lint:nofix": "vue-cli-service lint --no-fix",
     "test:unit": "vue-cli-service test:unit --testPathPattern",
     "test:unit-silent": "vue-cli-service test:unit --silent",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand",
-    "lernaupdate": "node_modules/.bin/lernaupdate"
```

Comment:
I removed this npm script, and dev dependency `lerna-update-wizard` below, because they caused more problems than they solved.